source/beamline/SGM/monochromator/SGMGratingSupport.h

```diff
@@ -0,0 +1,140 @@
+#ifndef SGMGRATINGSUPPORT_H
+#define SGMGRATINGSUPPORT_H
+
+/*!
+  * Namespace containing enumerators and functions relating to the grating translation
+  * on the SGM Beamline
+  */
+namespace SGMGratingSupport {
+
+	/*!
+	  * Enumerates the different selections for the grating translation.
+	  */
+	enum GratingTranslation {
+		UnknownGrating = -1,
+		LowGrating = 0,
+		MediumGrating = 1,
+		HighGrating = 2
+	};
+
+	/*!
+	  * Spacing value for each of the grating translations used in the calculation
+	  * of the energy produced.
+	  * \param translationSelection ~ The grating whose spacing value is to be
+	  * returned.
+	  */
+	inline static double spacing(GratingTranslation translationSelection)
+	{
+		switch(translationSelection) {
+		case LowGrating:
+			return 1.689233e-6;
+		case MediumGrating:
+			return 9.163583e-7;
+		case HighGrating:
+			return 5.903703e-7;
+		default:
+			return 0;
+		}
+	}
+
+	/*!
+	  * Correction value used in the calculation of the energy produced by a given
+	  * grating translation.
+	  * \param translationSelection ~ The grating whose correction value is to be
+	  * returned.
+	  */
+	inline static double c1(GratingTranslation translationSelection)
```

Comment:
Are there more verbose names you could give these guys?  I know they were probably called c1 and c2, but if they have a contextual name, I think that would benefit everyone (including our future selves when coming back to this code in the future).